sdk/include/psdk/dwmapi.h

```diff
@@ -41,7 +41,7 @@ static const UINT c_DwmMaxAdapters = 16;
 typedef ULONGLONG DWM_FRAME_COUNT;
 typedef ULONGLONG QPC_TIME;
 
-enum DWMWINDOWATTRIBUTE {
+enum _DWMWINDOWATTRIBUTE {
```

Comment:
You need typedef then.
Side note: check also the official headers. Some definitions appear to not have the typedef thing (maybe because dwmapi header was meant for c++ primaryily, and then on c you would need to repeat "enum ..." everytime?)